### PR TITLE
docs: streamline issue and pull request templates

### LINKED ISSUE #169: Review existing pull requests and issue templates

### PR DESCRIPTION
**Pull request by ejfox**
Fixes #169

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
-### Summary
-Tell us about your problem. What operating system are you using? How big was your dataset?
+## Summary
+What are you thinking?
```
Comment (newsroomdev) [inline .github/ISSUE_TEMPLATE.md:2]: Maybe, "What's on your mind?" or "What are you thinking about?" There's a part of me that's reading that can read this in a more aggressive way if I was a first-timer, e.g. "WHAT are you thinking?!"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ Fixes #[INSERT GITHUB ISSUE NUMBER HERE]
 
 **Summary**
 
-Tell us about your proposed changes.
+Tell us what you are thinking.
 
 **Proposed changes**
```
Comment (newsroomdev) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
### Proposed changes**
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -11,6 +11,4 @@ Provide a few more details like what files were modified, added or removed.
 * What was modified:
 * What was added:
 * What was removed:
-* Next actions:
```
Comment (newsroomdev) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: Maybe one more section for how to best review?

```suggestion
## How to review
```

**Review by newsroomdev — COMMENTED** [review]
Had a couple of more thoughts I wanted to jot down

**Review by newsroomdev — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (ejfox) [reply .github/PULL_REQUEST_TEMPLATE.md]: 👍

**Pull request merged**